### DISCUSSION TITLE
Does M$ somehow involved in Pulsar or addons development?

### DISCUSSION THREAD

**Discussion by Flashwalker** [pulsar-edit/.github#82] [Pulsar Support] [answered]
Does Microsoft have anything to do with Pulsar or addon development?
Are M$ repositories or something in use?

**Comment by confused-Techie** [comment, marked as answer]
Microsoft has no active role or participation within Pulsar at all. 

But it is possible that some of our repos have some sort of dependency with a repo from Microsoft, but as far as I'm aware the only integration we have with Microsoft is that we produce binaries to be installable on Windows systems.
